### ISSUE TITLE
[Bug]: worker_threads not work.

### ISSUE THREAD

**Issue by pool683** [electron/electron#30797]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Electron Version

15.0.0-alpha.10

### What operating system are you using?

Other Linux

### Operating System Version

Linux fedora 5.13.12-200.fc34.x86_64

### What arch are you using?

x64

### Last Known Working Electron version

14.0.0

### Expected Behavior

Should work without error.

### Actual Behavior

In browser console:
```
VM111 renderer_init:81
TypeError: SharedArrayBuffer is not a constructor
    at VM115 worker:96
    at NativeModule.compileForInternalLoader (VM10 loaders:312)
    at nativeModuleRequire (VM10 loaders:341)
    at VM114 node:worker_threads:11
    at NativeModule.compileForInternalLoader (VM10 loaders:312)
    at NativeModule.compileForPublicLoader (VM10 loaders:252)
    at loadNativeModule (VM92 helpers:40)
    at Module._load (VM89 loader:805)
    at Function.c._load (VM108 asar_bundle:5)
    at Function.o._load (VM111 renderer_init:29)
```

### Testcase Gist URL

https://gist.github.com/clavin/0b9f10737f395d36185c71b67c925a49

### Additional Information

[index.html]
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Hello World!</title>
    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'unsafe-inline';" />
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>
```

[main.js]
```
const { app, BrowserWindow } = require('electron')
const path = require('path')

function createWindow () {
  const win = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      preload: path.join(__dirname, 'preload.js')
    }
  })

  win.loadFile('index.html')
}

app.whenReady().then(() => {
  createWindow()

  app.on('activate', () => {
    if (BrowserWindow.getAllWindows().length === 0) {
      createWindow()
    }
  })
})

app.on('window-all-closed', () => {
  if (process.platform !== 'darwin') {
    app.quit()
  }
})
```

[preload.js]
`require('worker_threads');`

**Comment by clavin** [comment]
I was able to reproduce this error using this gist as an automated test: https://gist.github.com/clavin/0b9f10737f395d36185c71b67c925a49

**Comment by MarshallOfSound** [comment]
This has been fixed by @codebytere in 15.0.0-beta.2 and higher 🥳 https://github.com/electron/electron/releases/tag/v15.0.0-beta.2
